.github/workflows/publish-docker.yml

```diff
@@ -37,7 +34,6 @@ jobs:
       - name: Build and push Docker image
         uses: docker/build-push-action@a66e35b9cbcf4ad0ea91ffcaf7bbad63ad9e0229
         with:
-          context: .
           push: true
           tags: ${{ steps.meta.outputs.tags }}
```

Comment:
You should tag the docker images according the `git tags` like it is for example here: https://github.com/fluxcd/helm-controller/blob/main/.github/workflows/release.yml

Reply:
See next.